Sources/Extensions/UIKit/UITableViewExtensions.swift

```diff
@@ -181,6 +181,18 @@ public extension UITableView {
         return indexPath.section < self.numberOfSections && indexPath.row < self.numberOfRows(inSection: indexPath.section)
     }
 
+    /// SwifterSwift: Safely scroll to possibly invalid IndexPath
+    ///
+    /// - Parameters:
+    ///   - indexPath: Target IndexPath to scroll to
+    ///   - scrollPosition: Scroll position
+    ///   - animated: Whether to animate or not
+    public func safeScrollToRow(at indexPath: IndexPath, at scrollPosition: UITableViewScrollPosition, animated: Bool) {
+        if isValidIndexPath(indexPath) {
```

Comment:
Thanks for this submission @setoelkahfi 👍 Unfortunately, we don't allow dependencies between two SwifterSwift extensions

Reply:
Oh, I'll change it